src/OVAL/oval_generator.c

```diff
@@ -106,7 +104,12 @@ char *oval_generator_get_product_version(struct oval_generator *generator)
 
 char *oval_generator_get_schema_version(struct oval_generator *generator)
 {
-	return generator->schema_version;
+	return (char *) oval_generator_get_core_schema_version(generator);
```

Comment:
This nasty const_cast deserves a comment. I know why it's being done but another code reader may not.